### DISCUSSION TITLE
Getting 502 Bad Gateway nginx/1.25.1 when creating a classroom and assignment

### DISCUSSION THREAD

**Discussion by sukhjitsehra** [community/community#139388] [GitHub Education]
### Select Topic Area

Question

### Body

I am trying to create a classroom. I get 502 Bad Gateway nginx/1.25.1, but it creates a class, too. 

Then, I try to create an assignment, it shows 502 Bad Gateway nginx/1.25.1 again but not able to proceed, further. 

Any suggestions?

**Comment by Ranojitdas** [comment]
To fix the 502 Bad Gateway issue:

Increase Nginx Timeouts: In nginx.conf, increase timeouts:

```
bash
proxy_read_timeout 300;
proxy_connect_timeout 300;
proxy_send_timeout 300;
```
Check Upstream Server: Ensure your backend is working and responding properly.

Logs: Check Nginx and backend logs for more details on the issue.

This should help pinpoint or resolve the problem!

**Comment by sukhjitsehra** [comment]
Thanks for your inputs, where I can find those settings to Increase Nginx Timeouts (i.e. n nginx.conf) in my gihub classroom account information?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @sukhjitsehra !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the `GitHub Education` category. I've gone ahead and moved it for you. Good luck!

**Reply by sukhjitsehra** [reply]
Thanks for moving it. Yet, seeing the same error. 

Neither am I able to create any assignment nor able to use the previous class to create an assignment.
